servicex_codegen/post_operation.py

```diff
@@ -92,7 +93,7 @@ def post(self):
                 zip_data = self.stream_generated_code(generated_code_result)
                 # code gen transformer returns the default transformer image mentioned in
                 # the config file
-                transformer_image = current_app.config.get("TRANSFORMER_SCIENCE_IMAGE")
+                transformer_image = os.environ.get('TRANSFORMER_SCIENCE_IMAGE')
```

Comment:
I would stick with one place that reads the environment var and puts it in the app config and then use app config here.

It will surprise someone later that the property in app.config isn't used here!

Reply:
I have removed the DS store and added the .gitignore, for the app config thing, for some reason it does not work as intended, we have triend over 10 alpha version tests to check this thing, os.environ.get works but current_app.config does not, further, in the __init__.py the equation has to be there to set the value from os.environ to the app config. Although it does not make sense from a code perspective to do this, it does not work otherwise